### ISSUE TITLE
Input Dialog allow empty input

### ISSUE THREAD

**Issue by Kiwifed0r** [afollestad/material-dialogs#452]
First of all, nice work!

I created an input dialog but the onInput callback is only called when the EditText is not empty. I had a look through the code and I found that it is supposed to be like that. For now, I just added another callback for onPositive and just ignore the onInput callback. Is there some option I missed that allows empty input and still fires the onInput callback?

Regards

**Comment by afollestad** [comment]
https://github.com/afollestad/material-dialogs/releases/tag/v0.7.2.1

**Comment by Ph1b** [comment]
Now its getting confusing. How can I manually disable the Okay button when text is emmpty? Was happy with how it worked, no my apps crash.

My workaround is to have a String declared before the builder, then in onInput, update that string with the text. 

Before the builder I declared a InputCallback and before that a button, I initialized after the .show() and have the inputCallback enabled / disable the button.

 Then I use the buttonCallbacks onPositive to get the text when I need him. Oh and I had to set alwaysCallInputCallback, so I can disable the button when text changes and is empty.

Anyway, thats really not the easy way ;-) Maybe just pass a `boolean allowEmptyText` to the input?
